# Service that can be used to get pose information about the EE using MoveIt.

---
geometry_msgs/Pose pose
bool success
string message
